include/circt/Dialect/FIRRTL/FIRRTLExpressions.td

```diff
@@ -1045,18 +1045,18 @@ def StringConstantOp : FIRRTLOp<"string", [Pure, ConstantLike]> {
   let assemblyFormat = "$value attr-dict";
 }
 
-def BigIntConstantOp : FIRRTLOp<"bigint", [Pure, ConstantLike]> {
-  let summary = "Produce a constant bigint value";
+def IntegerConstantOp : FIRRTLOp<"integer", [Pure, ConstantLike]> {
```

Comment:
Maaaybe this could be `FIntegerConstantOp` for symmetry with the type name of these constants?